docker-bake.hcl

```diff
@@ -275,7 +287,7 @@ target "virtual-postgres" {
         dockerfile = "src/images/postgres.Dockerfile"
         inherits = [
                 "virtual-default",
-                "virtual-platforms",
+                "virtual-platforms-postgres",
```

Comment:
I would just drop the `virtual-platforms` from the `inherits` and directly set the `platforms` value here. Or is there a reason to extract it into a shared inheritable set?

Reply:
no reason to, added directly.